docs/src/token-2022/extensions.mdx

```diff
@@ -1347,6 +1347,10 @@ A usable example program exists at
 Token-2022 uses this example program in tests to ensure that it properly uses
 the transfer hook interface.
 
+The example program and the interface are powered by the
+[spl-tlv-account-resolution](https://github.com/solana-labs/solana-program-library/tree/master/libraries/tlv-account-resolution/README.md)
+library, which is explained a bit more below.
+
 #### How to use
```

Comment:
There's a lot of copy-pasta between all these sections -- how about just linking these to the sections in the README with one sentence?